packages/components/inputs/radio-input/src/icons/create-styled-icon.js

```diff
@@ -0,0 +1,100 @@
+/*
+  THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+
+  This file is maintained by the 'scripts/generate-icon-exports.js' script.
+*/
+import PropTypes from 'prop-types';
+import invariant from 'tiny-invariant';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { withTheme } from 'emotion-theming';
+import { customProperties as vars } from '@commercetools-uikit/design-system';
+
+const iconSizes = {
+  small: 12,
+  medium: 16,
+  big: 24,
+};
+
+export const getSizeStyle = (size) => {
+  switch (size) {
+    case 'scale':
+      return `
+        &:not(:root) {
+          width: 100%;
+          height: auto;
+        }
+      `;
+    case 'small':
+    case 'medium':
+    case 'big':
+      return `
+        width: ${iconSizes[size]}px;
+        height: ${iconSizes[size]}px;
+      `;
+    default:
+      return `
+        width: ${iconSizes.big}px;
+        height: ${iconSizes.big}px;
+      `;
+  }
+};
+
+const capitalize = (s) => s[0].toUpperCase() + s.slice(1);
```

Comment:
although this is simple, I'd rely on `lodash.capitalize` more

Reply:
Not related to this PR